wire-ios/Wire-iOS Tests/ConversationOptionsViewControllerTests.swift

```diff
@@ -122,7 +122,11 @@ final class ConversationOptionsViewControllerTests: BaseSnapshotTestCase {
     func testThatItRendersGuestsScreenWhenAllowGuestsIsDisabled() {
         // GIVEN
         let config = MockOptionsViewModelConfiguration(allowGuests: false)
-        let viewModel = ConversationGuestOptionsViewModel(configuration: config, conversation: mockConversation.convertToRegularConversation(), userSession: mockUserSession)
+        let viewModel = ConversationGuestOptionsViewModel(
+            configuration: config,
+            conversation: mockConversation.convertToRegularConversation(),
+            userSession: mockUserSession
+        )
```

Comment:
This repeats a lot and if makes it quite time consuming if you add a new parameter in 30 tests.
My solution to this is usually to add a helper func and just optionally pass parameters that differ:
```swift
private func makeViewModel(config: ViewModelConfiguration) -> ConversationGuestOptionsViewModel {
        ConversationGuestOptionsViewModel(
            configuration: config,
            conversation: mockConversation.convertToRegularConversation(),
            userSession: mockUserSession
        )
}
```

Reply:
Done!